UnitedIncome/components
# Font Size Change

### Description
> Please provide a brief description of your changes below.

Adjusts the `rem` sizing based on an internal discussion.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at the wonderful new mobile sizes.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
